core/src/main/scala/chisel3/experimental/hierarchy/core/Lookupable.scala

```diff
@@ -294,7 +300,7 @@ object Lookupable {
     def instanceLookup[A](that:   A => B, instance:   Instance[A]):   C = that(instance.proto)
   }
 
-  implicit def lookupInstance[B <: BaseModule](implicit sourceInfo: SourceInfo) =
+  implicit def lookupInstance[B <: BaseModule](implicit sourceInfo: SourceInfo): Simple[Instance[B]] =
```

Comment:
I'm wondering if we should include this `Simple` alias or just use `Aux` here making the type ascription `: Aux[Instance[B], Instance[B]]`. It's not _that_ bad and reduces the number of type aliases which may be more clear. Any thoughts?

Reply:
I'm fine with the extra alias. It could be useful for users who define their own `Lookupable`  instances.